ccpp/data/GFS_typedefs.meta

```diff
@@ -4638,6 +4684,56 @@
   type = character
   kind = len=3
   active = (control_for_stochastic_land_surface_perturbation /= 0)
+[n_var_spp]
+  standard_name = number_of_spp_schemes_perturbed
+  long_name = number of spp schemes perturbed
+  units = count
+  dimensions = ()
+  type = integer
+[spp_prt_list]
+  standard_name =magnitude_of_perturbations_for_spp
+  long_name = magnitude of perturbations for spp
+  units = variable
```

Comment:
I think that this needs to be "units = 1". "Units = variable" may cause trouble in the future if not now. I realize that this was probably following the land surface perturbations, but we'll probably need to clean metadata associated with that up in the future.

Reply:
Updated.  Thanks!

Reply:
Same here. The unit is still "variable"

Reply:
Fixed!  Thanks!